calculator/lexer.py

```diff
@@ -11,8 +72,85 @@ def lex(s):
       * "*" - Multiplication
       * "+" - Addition
       * "-" - Subtraction
+      * "/" - Division
     * The following grouping symbols are supported:
       * "("
       * ")"
     """
-    return []
+
+    # Defining char_before and char_after
+    char_before = None
+    char_after = None
+    # Partial_num is the var that stores my nums until I have to append it to my tokens list
+    partial_num = None
+    # List that stores the tokens
+    tokens = []
+    # Stripping the input line of spaces
+    stripped_line = line
```

Comment:
This is no longer `stripped_line`, can you rename to something more appropriate or just remove `stripped_line` altogether?  I think it's probably sensible to `trim()` the line fwiw if you wanted to do that.

Reply:
wrong. I am going to strip() the code, so it would be stripped_line.